sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/IPAddressOrRange.Serialization.cs

```diff
@@ -10,20 +10,20 @@
 
 namespace Azure.ResourceManager.CosmosDB.Models
 {
-    public partial class CosmosDBIPAddressOrRange : IUtf8JsonSerializable
+    public partial class IPAddressOrRange : IUtf8JsonSerializable
```

Comment:
I wonder why these types change their name here without a configuration change.
Are these expected?
Quite a few of them are breaking changes.